src/wanderer/guzzlr.ts

```diff
@@ -40,12 +40,11 @@ function acceptGuzzlrQuest(locationSkiplist: Location[]) {
       !(get("garbo_prioritizeCappingGuzzlr", false) && Guzzlr.haveFullPlatinumBonus())
     ) {
       Guzzlr.acceptPlatinum();
-    } else if (Guzzlr.canGold() && (Guzzlr.haveFullBronzeBonus() || !Guzzlr.haveFullGoldBonus())) {
-      // if gold is not maxed, do that first since they are limited per day
-      Guzzlr.acceptGold();
-    } else {
-      // fall back to bronze when can't plat, can't gold, or bronze is not maxed
+    } else if (!Guzzlr.haveFullBronzeBonus() || !Guzzlr.canGold()) {
```

Comment:
As per our discussion in discord, this may not be optimal wrt capping the guzzlr progress asap.
I'd do something like `else if ((!Guzzlr.haveFullBronzeBonus() && get("garboGuzzlrPrioritizeBronze", false)) || !Guzzlr.canGold())` where users can choose to set that preference if they want to max out their bronze bonus asap